### DISCUSSION TITLE
HELP

### DISCUSSION THREAD

**Discussion by rodrigo-mtto** [lowag/libremaint#2] [Q&A] [answered]
HELLO! .
failed connecting to database (2002) php_network_getaddrinfo fail: temporary failure in name resolution

**Comment by lowag** [comment, marked as answer]
Hello,
In the 'libremaint/config/lm-config.php' file at the end : define('HOST','your_database_host'); the value probably wrong. Try to your IP address belong to your database.
 That's why the database is unreachable.
B.R.
Antal

**Reply by rodrigo-mtto** [reply]
Thank you so much!!!

now the PHP application is running on the WEB but without template ...

I know my questions are basic but I have no experience in this area :( :( :( :( 

thanks in advance

**Reply by lowag** [reply]
Unfortunately there is no documentation at this stage. Maybe I should make somekind of basic template with data ...

**Comment by rodrigo-mtto** [comment]
Hello again.
Excuse the insistence.

I would like to know if you can advise me, the idea is to give you access to the server in order to see the problem I am having with the visual and the bugs.

grateful beforehand.

RTR

**Reply by lowag** [reply]
In the file 'libremaint/config/lm-settings.php' somewhere at the row 63 there are two rows: define('LM_DEBUG',TRUE); and define('LM_DEBUG_LOG',TRUE); If both are TRUE then must be a file 'libremaint/lm-debug.log'. Check it.
